js/script.js

```diff
@@ -1,5 +1,33 @@
-const table = document.getElementById('assignmentsTable');
-let baseRow = document.getElementById("baseRow");
+const table = document.querySelector('#assignmentsTable');
+const buttonInsert = document.querySelector('.buttonInsert');
+var allInputs = document.querySelectorAll('input');
+var rowNo = 2;
+buttonInsert.addEventListener('click', function () {
+    var row = table.insertRow(rowNo)
+    var cell1 = row.insertCell(0);
+    var cell2 = row.insertCell(1);
+    var cell3 = row.insertCell(2);
+    var cell4 = row.insertCell(3);
+    var cell5 = row.insertCell(4);
+    var cell6 = row.insertCell(5);
+    var cell7 = row.insertCell(6);
+
+    cell1.innerHTML = document.querySelector('.assignmentName').value;
+    cell2.innerHTML = document.querySelector('.className').value;
+    cell3.innerHTML = document.querySelector('.dateinput').value;
+    cell4.innerHTML = document.querySelector('.timeinput').value;
+    cell5.innerHTML = document.querySelector('.priorityInput').value;
+    cell6.innerHTML = document.querySelector('.estimatedTimeInput').value;
+    cell7.innerHTML = document.querySelector('.status').value;
```

Comment:
We'd probably like to use `innerText` rather than `innerHTML` to prevent any cross-site scripting (XSS) hacks.

```suggestion
    cell1.innerText = document.querySelector('.assignmentName').value;
    cell2.innerText = document.querySelector('.className').value;
    cell3.innerText = document.querySelector('.dateinput').value;
    cell4.innerText = document.querySelector('.timeinput').value;
    cell5.innerText = document.querySelector('.priorityInput').value;
    cell6.innerText = document.querySelector('.estimatedTimeInput').value;
    cell7.innerText = document.querySelector('.status').value;
```